testassets/InteropTestsGrpcWebWebsite/Tests/package.json

```diff
@@ -6,8 +6,8 @@
     "test": "jest test"
   },
   "dependencies": {
-    "puppeteer": "^11.0.0",
-    "jest": "^27.3.1",
-    "jest-puppeteer": "^6.0.0"
+    "puppeteer": "^13.3.2",
```

Comment:
Not super relevant but do we want to pin these to the patch version for more consistency in the build?

Not sure if this repo is set up with dependabot which would defeat the purpose but just a thought

Reply:
Doesn't the lock file pin dependencies for the build?

I'm not a node expert but the ^ syntax is used in package.json files in aspnetcore repo.

Reply:
Yes, it does! Missed that there were lock files in here. 😅